docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/trust-login-source.md

```diff
@@ -0,0 +1,94 @@
+---
+id: trust-login-source
+title: Trust Login Source
+sidebar_label: Trust Login
```

Comment:
I think all instances of "Trust Login" might need to be changed to "TrustLogin". Check with the product manager or development lead.

Reply:
@jpipkin1 confirmed with @draval-sumo . The correct naming for the integration is "Trust Login".